### PR TITLE
feat: integration organization to object

### PR DESCRIPTION
**Pull request by MNAlfiansyah**
## Changes

- integration organization to object

## Evidence
title: Form Object
project: Jabar Super Apps
participants: @rayfajars @johndy2742 @ristianaditya @MNAlfiansyah

### CODE REVIEW

File: dashboard/components/Object/Form/index.vue [hunk 1]
```diff
@@ -255,50 +291,75 @@
...
+          village_id: responseObjectData?.village_id,
+          google_map: responseObjectData?.google_map,
         }
 
-        state.dataUrlImage = responseData.logo
-        state.dataUrlImageMultiple = responseData.banner
+        
+
+        state.dataUrlImage = responseObjectData.logo
+        // state.dataUrlImageMultiple = responseObjectData.banner
```
Comment (rayfajars) [inline dashboard/components/Object/Form/index.vue:339]: is used?

File: dashboard/components/Object/Form/index.vue [hunk 2]
```diff
@@ -486,7 +561,15 @@
     const dataAreExist = []
     if (imageMultipleLength >= 1) {
       for (let i = 0; i < imageMultipleLength; i++) {
-        if (imageMultiple[i].url == '') {
+        if (imageMultiple[i].url === '') {
```
Comment (rayfajars) [inline dashboard/components/Object/Form/index.vue:564]: same thing with truthly js

**Review by rayfajars — APPROVED** [review]

File: dashboard/components/Object/Form/index.vue [hunk 3]
```diff
@@ -34,6 +34,17 @@
...
+        </div>
+        <div class="mt-5">
+          <BaseDropdown
+            name="organization_id"
+            type="text"
+            label="Organisasi"
+            placeholder="Pilih Organisasi"
+            :data-dropdown="state.dataOrganisasi"
+            :disabled="state.isFormDisabled"
           />
```
Comment (Ibwedagama) [inline dashboard/components/Object/Form/index.vue:48]: hello @MNAlfiansyah how does this component `ObjectForm` know which data is being selected by <BaseDropdown> component? isn't BaseDropdown only store selected inside it's component? 🤔

Comment (MNAlfiansyah) [reply dashboard/components/Object/Form/index.vue:48]: the data has set on the v-model from the form, this is the code. 
this code are for set the value of v-model on each component, the v-model is define by the component name
Screenshot 2023-11-15 at 17 39 45

**Review by marsellavaleria19 — APPROVED** [review]

File: dashboard/components/Object/Form/index.vue (hunk 1 above)
Comment (MNAlfiansyah) [reply dashboard/components/Object/Form/index.vue]: thanks for notice on this one kang, it used i already update [cf2583b](https://github.com/jabardigitalservice/event-management-system/pull/84/commits/cf2583baf714a1e370466ec7c5cd3082f25ff0e9)

File: dashboard/components/Object/Form/index.vue (hunk 2 above)
Comment (MNAlfiansyah) [reply dashboard/components/Object/Form/index.vue]: updated kang [cf2583b](https://github.com/jabardigitalservice/event-management-system/pull/84/commits/cf2583baf714a1e370466ec7c5cd3082f25ff0e9)

**Pull request merged**